src/Sylius/Bundle/ApiBundle/Validator/Constraints/ShopUserVerificationState.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/** @experimental */
+final class ShopUserVerificationState extends Constraint
```

Comment:
To be consistent with other constraints, it might look like:
```suggestion
final class ShopUserNotVerified extends Constraint
```